File: Aplicacion Windows/TFG_Windows/TFG/BD/PhysicalActivityDTO.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;
using System.Data.SqlTypes;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;
using TFG.Enumerados;
using TFG.Fecha;

namespace TFG.BD
{
    public class PhysicalActivityDTO
    {
        public long id { get; set; }

        public long usuario_id { get; set; }
        public string fecha { get; set; }

        public string tipo { get; set; }

        public int duracion { get; set; }
    }
}

File: Aplicacion Movil/MedMonitor/app/src/main/java/Models/PhysicalActivityDTO.kt
package Models

import Enums.PhysicalActivityTipe
import java.io.Serializable

data class PhysicalActivityDTO(
    val id: Long? = null,
    val usuario_id: Long,
    val fecha: String,  // YYYY-MM-DD
    val tipo: PhysicalActivityTipe,
    val duracion: Int   // En minutos
):Serializable
